### PR TITLE
[Merged by Bors] - feat: the bicentralizer of a commutative set is commutative

### PR DESCRIPTION
**Pull request by j-loreaux**
This shows that the bicentralizer (a.k.a. bicommutant) of a commutative set is commutative. Moreover, since `s ⊆ s.centralizer.centralizer`, if `s` is a commutative set, then `closure s ≤ centralizer (centralizer s)` for various subobject closures. Consequently, we obtain simplified proofs that if `s` is commutative, then so is `closure s`.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Group/Submonoid/Membership.lean [hunk 1]
```diff
@@ -6,6 +6,7 @@ Amelia Livingston, Yury Kudryashov
 -/
 import Mathlib.Algebra.FreeMonoid.Basic
 import Mathlib.Algebra.Group.Submonoid.MulOpposite
+import Mathlib.GroupTheory.Submonoid.Centralizer
```
Comment (j-loreaux) [inline Mathlib/Algebra/Group/Submonoid/Membership.lean:9]:
```suggestion
```

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge

**Comment by j-loreaux** [comment]
Sorry, I think @ADedecker wanted to make some comments or contributions here.

**Comment by jcommelin** [comment]
Ok, I've labeled it WIP for the time being.

**Comment by j-loreaux** [comment]
@jcommelin I think this is ready again.

**Comment by riccardobrasca** [comment]
Thanks!

bors merge

**Pull request closed without merging**